server/services/gameplay.go

```diff
@@ -0,0 +1,51 @@
+package services
+
+import (
+	"context"
+
+	controller "github.com/MettyS/checkers/server/controller"
+	pb "github.com/MettyS/checkers/server/generated"
+)
+
+// GameplayServer server for gameplay service
+type GameplayServer struct {
+	pb.UnimplementedGameplayServiceServer
+}
+
+func convertMoveRequestInward(req *pb.MoveRequest) *controller.MoveRequest {
+	domMoveRequest := &controller.MoveRequest{}
```

Comment:
As mentioned in another file, I think we should use values everywhere it's possible, so refs/pointers signal something else is going on.

Reply:
Done.